### DISCUSSION TITLE
Single-platform chat identity recommendations

### DISCUSSION THREAD

**Discussion by OrO-c** [Taewan-P/gpt_mobile#94] [Ideas]
When using a single platform, please avoid displaying "Powered by xxxx" in the response. Users can already clearly see which platform is being used for the current session from the homepage, and displaying this information may seem redundant. Simplifying the interface can help enhance the user experience.![Screenshot_20241030194909.jpg](https://github.com/user-attachments/assets/a8fbd614-b9bb-40d5-a199-7ed862d4bce7)

**Comment by Taewan-P** [comment]
This is intended due to the [OpenAI's Brand Guidelines](https://openai.com/brand/). The app must include what they are using.
